assets/css/site.css

```diff
@@ -54,6 +58,94 @@ h1 {
 .container .share-bottom {
   margin: 15px 0px 15px 0px;
 }
+
+.container .button {
+  z-index: -1;
+  border-bottom: 0;
+  top: 0;
+  right: 0;
+  background-color: transparent;
+  font-family:Arial, Helvetica, sans-serif;
+  font-size: 0.8em;
+  display: inline-block;
+  padding: 12px 16px 100% 100%;
+  border: none;
+  min-width: 100%;
+  color: transparent;
+  cursor: default;
+  text-transform: uppercase;
+  position: fixed;
+  font-weight: 700;
+}
+
+.container .button-x {
+  border: none;
+  border-bottom: 0;
+  position: absolute;
+  right: -35px;
+  top: -30px;
+  width: 30px;
+  height: 30px;
+  background-color: #222;
+  color: #fff;
+  font-family:Arial, Helvetica, sans-serif;
+  font-size: 0.8em;
+  font-weight: 600;
+  line-height: 30px;
+  text-align: center;
+  border-radius: 50%;
+}
+.popup {
+  z-index: 1;
+  opacity: 0;
+  pointer-events: none;
+  transition: 0.4s;
+
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.705);
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 16px;
+}
+
+.popup-inner {
```

Comment:
Is this `.popup-inner` used somewhere? Do we really need it? If so, when do we use it? Can we have a more descriptive name?